src/common/components/ui/formik/types.ts

```diff
@@ -101,7 +101,6 @@ export interface UploadFieldProps extends FieldProps {
   dropzoneProps?: Omit<DropzoneProps, 'onDropRejected'> & {
     onDropRejected?: (helpers: FieldHelperProps<string>) => DropFileEventHandler;
   };
-  transformValue?: (key: string) => void;
```

Comment:
This prop had only been passed from OCM's UploadSSH field, but it was recently removed.